app/_src/guides/progressively-rolling-in-strict-mtls.md

```diff
@@ -5,7 +5,42 @@ title: Progressively rolling in strict mTLS
 The [MeshTLS](/docs/{{ page.version }}/policies/meshtls/) policy allows you to gradually migrate services to mutual TLS without dropping a packet.
 
 ## Prerequisites
-- Completed [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/) to set up a zone control plane with demo application
+- Completed [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/) to set up a zone control plane with demo application.
+
+## Basic setup
+
+In order to be able to fully utilize MeshTLS policy you need to enable [Mutual TLS](/docs/{{ page.version }}/policies/mutual-tls/) (mTLS), and we can do it with `builtin` CA backend by executing:
```

Comment:
I thought that we're not supposed to use `we` to indicate `us` and the `user` rather you should use `you`. Do I remember it correctly?

```suggestion
In order to be able to fully utilize MeshTLS policy you need to enable [Mutual TLS](/docs/{{ page.version }}/policies/mutual-tls/) (mTLS), and you can do it with `builtin` CA backend by executing:
```

Reply:
In this documentation, I found that `we` and `you` for several times. Are there any norms for this?

Reply:
Yeah. I don't know. There's a Vale rule that warns you about it. Both sound weird sometimes but probably "you" is better. Kubernetes uses "you".